packages/react-priority-overflow/src/components/Overflow.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { OverflowContext } from '../overflowContext';
+import { defaultUpdateVisibilityCallback, useOverflowContainer } from '../useOverflowContainer';
+import { useMergedRefs } from '@fluentui/react-utilities';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import type { OnUpdateOverflow, OverflowGroupState, ObserveOptions } from '@fluentui/priority-overflow';
+
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    flexWrap: 'nowrap',
+    minWidth: 0,
+    ...shorthands.overflow('hidden'),
+  },
+
+  vertical: {
+    flexDirection: 'column',
+  },
+});
+
+export interface OverflowProps
+  extends React.HTMLAttributes<HTMLDivElement>,
+    Omit<ObserveOptions, 'onUpdateOverflow' | 'onUpdateItemVisibility'> {}
```

Comment:
I would rather use `Pick` to ensure that new features added to `ObserveOptions` will not accidentally leak.